src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -96,6 +96,14 @@ module sonic-port{
 						pattern "on|off";
 					}
 				}
+
+				leaf tpid {
```

Comment:
1. Add description field to describe what this attribute is about
2. move to common so both port and LAG can share to avoid duplication

Reply:
we will not follow the open config model for this... for simplicity handling in config DB.

Reply:
In the future when stateDB validation method is agreed, we can come back to this and enhance it to check capability as part of the yang model validation for TPID attribute